.travis.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         - yarn run build-storybook
         - yarn run serveAndRunUiTests
     - stage: deploy
-      if: (tag = ^v) AND (branch = master)
+      if: (tag =~ ^v) AND (branch = master)
```

Comment:
The [Git flow cheatsheet](https://danielkummer.github.io/git-flow-cheatsheet/) that I referred in the strategy spike recommend `release/vX.Y.Z` as branch name, hence I suggest `tag=~ ^release\/` because otherwise you are going to catch branch names that simply start with `v`.

Reply:
My bad, I misread!

You should:
1. create a branch `release/vX.Y.Z` (update package.json with new version)
2. merge on master
3. tag the merge commit